packages/components/breadcrumbs/src/breadcrumbs.stories.ts

```diff
@@ -29,7 +31,7 @@ export const Basic: Story = {
     breadcrumbs: html`
       <a href="javascript:void(0)">Lorem</a>
       <a href="javascript:void(0)">Ipsum</a>
-      <span>Dolar</span>
+      <a href="javascript:void(0)">Dolar</a>
     `
   }
```

Comment:
So breadcrumbs can be moved from `in progress` to `components`? :)

<img width="289" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/d8d38848-ed44-4336-b35f-f571ed3cda5c">

Reply:
I think we only should do this once we also add them to the documentation website. If teams are really eager to start using it we can let them, but if we wait to move it to components until the documentation is ready we might save our selves some work answering questions.